android/src/main/java/br/com/classapp/RNSensitiveInfo/view/Fragments/FingerprintAuthenticationDialogFragment.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (C) 2015 The Android Open Source Project
+ *
+ * Modifications copyright (C) 2018 Sowa Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+
+package br.com.classapp.RNSensitiveInfo.view.Fragments;
+
+import android.app.DialogFragment;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.SharedPreferences;
+import android.hardware.fingerprint.FingerprintManager;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v7.app.AppCompatActivity;
```

Comment:
Just a comment, the react native activity does not extend AppCompatActivity and when at line 143 you do the cast,
it will throw a ClassCastException. After a check, it seems, there is no need to use AppCompatActivity. 
I changed the cast to Activity and everything works well now.

Reply:
Great, thanks for noticing! I will publish a pull request with the fix.

Reply:
Fix is already in version 5.2.3:

```bash
yarn add react-native-sensitive-info@beta
```